### PR TITLE
Removes trailing slash from blog url and adds templates for issue and PR

### PR DESCRIPTION
**Pull request by st-jay**
## Connected issue: #16 

# Changes made
- Added an issue template with the blog checklist ( https://github.com/supertokens/main-website/blob/master/.github/ISSUE_TEMPLATE/blog.md )
  - Removed the webflow and node process related steps
- Added PR templates for new blog PRs ( this is the default template ) and for dev PRs
- Updated the blog listing page to remove trailing slash `/` from blog urls

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/dev.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+## Connected Issues
```
Comment (nkshah2) [inline .github/PULL_REQUEST_TEMPLATE/dev.md:1]: What would the experience of multiple PR templates look like on Github? (Try it out on a sample project and add screenshots if possible)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Related Issue
+
```
Comment (nkshah2) [inline .github/PULL_REQUEST_TEMPLATE.md:2]: Add some text explaining what this should be (single line or example is fine)

**Review by nkshah2 — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (st-jay) [reply .github/PULL_REQUEST_TEMPLATE.md:3]: Updated

File: .github/PULL_REQUEST_TEMPLATE/dev.md (hunk 1 above)
Comment (st-jay) [reply .github/PULL_REQUEST_TEMPLATE/dev.md:1]: The template for PRs adding a new blog will be the default template. To change to the `dev.md` template, when creating the PR, users will have to add a `template=dev.md` query param. I've added this explanation to the `CONTRIBUTING.md` file.

**Review by nkshah2 — APPROVED** [review]

**Pull request merged**